runtime/lua/vim/lsp/_inlay_hint.lua

```diff
@@ -238,9 +239,12 @@ api.nvim_set_decoration_provider(namespace, {
       return
     end
     local hints_by_client = bufstate.client_hint
-    api.nvim_buf_clear_namespace(bufnr, namespace, 0, -1)
 
     for lnum = topline, botline do
+      if bufstate.applied[lnum] == bufstate.version then
+        goto continue
+      end
```

Comment:
goto is afaik only available with luajit not with plain lua 5.1 and neovim targets plain lua 5.1 as well

Reply:
What do I do instead?